test/samples/solidity/latest_08.sol

```diff
@@ -385,3 +385,24 @@ contract Features_0819 {
         return a + b;
     }
 }
+
+interface IntEvents {
+    event X(uint a);
+}
+
+library LibEvents {
+    event X(uint a);
+}
+
+event X(uint a);
+
+contract Features_0822 {
+    event X(uint a);
+
+    function main() public {
+        emit IntEvents.X(1);
+        emit LibEvents.X(2);
+        emit X(3);
```

Comment:
This is a nit, not neccessary to fix. But I think this emit refers to the event defined in the contract, not the file-level event definition. So there is no emit that actually tests the file-level event def

Reply:
Addressed in 8fdc8af